### DISCUSSION TITLE
Can I include charge (or custom loss function) in training?

### DISCUSSION THREAD

**Discussion by turbosonics** [ACEsuit/mace#358] [Q&A] [answered]
This question is continuation of discussion from other post: https://github.com/ACEsuit/mace/discussions/344
But I think this topic worth a separate post, so I write here. 

I have AIMD data with charge per each atom. I was wondering if I can include charge as loss function to MACE, because I saw key for charges. But it seems that is not possible as of now. I don't see any charges_weight keyword for input configuration. I can only set weights for E, F, stress, virial, and dipole moment. 

In this case, is there any way to include charge per atom to MACE training? Or, is there any way to include custom loss function to MACE training?

**Comment by ilyes319** [comment, marked as answer]
Charges are not supported currently as output of MACE.

**Reply by turbosonics** [reply]
Thanks, but are there any plans to include the feature for charge or any custom loss functions in the future?

**Reply by ilyes319** [reply]
It is not a problem of loss function but of electrostatic modelling. And yes, we are currently working on that.

**Reply by turbosonics** [reply]
> but of electrostatic modelling.

Actually, I was hoping to test that... 

Anyway, thanks to let me know. I will wait for more MACE update in the future while train model with what I have.
